kansha/board/comp.py

```diff
@@ -161,6 +163,10 @@ def many_user_render(h, number):
 
         self.must_reload_search = False
 
+    def reload_save_template_comp(self):
+        self.save_template_comp = component.Component(SaveTemplateTask(self))
+        self.save_template_comp.on_answer(lambda v: self.reload_save_template_comp())
+
     def copy(self, owner, additional_data):
```

Comment:
Some informations need to be propagated through the components tree, for example labels are created at the board level and must be available to cards as they use it. These are stored in this dict to be used later.